primap2/pm2io/_conversion.py

```diff
@@ -62,6 +63,11 @@ def convert_unit_primap_to_primap2(unit: str, entity: str) -> str:
     # remove spaces for more flexibility in input units
     unit = unit.replace(" ", "")
 
+    # check if entity contains GWP information. If so discard
+    # not needed for PRIMAP1 entities but when using the function for data reading
```

Comment:
I don't like updating the name because it means changing code that uses the function (though it's not too much code). But it's also weird to have a function that does more general conversion and has primap1 as a special case while still having a primap1 conversion function. It could be a wrapper that does a format check and then calls the general function.

Reply:
We could use a new name, and keep the old name for compatibility. Doesn't cost anything to have
```python
convert_unit_primap_to_primap2 = convert_unit_to_primap2
```
essentially forever.

Reply:
I renamed the function without adding an alias. We don't have enough usage yet to justify adding potentially confusing aliases.